src/Compilers/Core/Portable/Operations/ControlFlowGraph.Region.cs

```diff
@@ -3,216 +3,215 @@
 using System;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Operations
+namespace Microsoft.CodeAnalysis.FlowAnalysis
 {
-    public partial class ControlFlowGraph
+    /// <summary>
+    /// Defines kinds of regions that can be present in a <see cref="ControlFlowGraph"/>
+    /// PROTOTYPE(dataflow): Split <see cref="ControlFlowRegion"/> and <see cref="ControlFlowRegionKind"/> into separate files.
```

Comment:
>PROTOTYPE(dataflow): [](start = 8, length = 20)

Consider moving out of the public doc comment.

Reply:
Actually, will address this in a follow-up commit/PR.